src/Miunie.Discord/CommandModules/ProfileCommand.cs

```diff
@@ -1,54 +1,59 @@
 using System.Threading.Tasks;
-using DSharpPlus.CommandsNext;
-using DSharpPlus.CommandsNext.Attributes;
+using Discord.Commands;
+using Discord.WebSocket;
 using Miunie.Core;
 using Miunie.Discord.Convertors;
 
 namespace Miunie.Discord.CommandModules
 {
-    public class ProfileCommand : BaseCommandModule
+    public class ProfileCommand : ModuleBase<SocketCommandContext>
     {
         private readonly EntityConvertor _entityConvertor;
+        private readonly MiunieUserConverter _userConverter;
         private readonly ProfileService _profileService;
 
-        public ProfileCommand(EntityConvertor entityConvertor, ProfileService profileService)
+        public ProfileCommand(EntityConvertor entityConvertor, ProfileService profileService, MiunieUserConverter userConverter)
         {
             _entityConvertor = entityConvertor;
             _profileService = profileService;
+            _userConverter = userConverter;
         }
 
         [Command("profile")]
-        public async Task ShowProfileAsync(CommandContext ctx, MiunieUser m = null)
+        public async Task ShowProfileAsync(SocketGuildUser user = null)
         {
+            var m = _userConverter.DiscordMemberToMiunieUser(user ?? (Context.User as SocketGuildUser));
```

Comment:
variable 'm' isn't very descriptive tbh - though I guess you know what is due to the context it is used in.